modules/ROOT/pages/afm-public-versus-private.adoc

```diff
@@ -31,7 +31,7 @@ The shortest interval at which you can schedule a test to be run is fifteen minu
 
 == Monitoring the Endpoints of Private APIs
 
-With a subscription to Anypoint Virtual Private Cloud (VPC), you can run workers in a virtual, private, and isolated network segment, rather than in a region in which resources are shared. A _private location_ is a namespace in an instance of VPC that is configured to allow workers to access your private network. AFM can create workers within that namespace that can run tests against APIs that are accessible only within the network that VPC is configured for. You can create multiple private locations in a single VPC.
+Private APIs are APIs with endpoints that are accessible only within your network. API Functional Monitoring lets you monitor private APIs if you have a subscription to Anypoint Virtual Private Cloud (VPC). With VPC, you can run workers in CloudHub in a virtual, private, and isolated network segment, rather than in a region in which resources are shared. A _private location_ is a namespace in an instance of VPC that is configured to allow workers to access your private network. AFM can create workers within that namespace that can run tests against APIs that are accessible only within the network that VPC is configured for. You can create multiple private locations in a single VPC.
```

Comment:
I would rephrase the last part of the sentence to something like this:

> A _private location_ is an AFM worker running in the VPC configured. AFM can create multiple workers within that vpc in order to run scheduled tests against APIs that are accessible only within the network that VPC is configured for. You can create multiple private locations in a single VPC.

Reply:
I replied in Slack, so that I could include a diagram.